packages/starlight/components/SidebarSublist.astro

```diff
@@ -43,6 +47,22 @@ interface Props {
     margin-top: 0.75rem;
   }
 
+  .sidebar-group-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    cursor: pointer;
+    user-select: none;
+  }
+
```

Comment:
One last thing, to display correctly for languages written right-to-left (Arabic or Hebrew for example), we should flip the caret 180deg so it points left in these scenarios:

```suggestion

  :global([dir='rtl']) .caret {
    transform: rotateZ(180deg);
  }
```